beaufort.md

```diff
@@ -0,0 +1,59 @@
+# Beaufort
+
+An uncluttered dark theme that borrows the 
+
+![Beaufort](images/beaufort_preview.png)
+
+## Settings
+
+```json
+{
+    // PowerShell Beaufort profile
+    "hidden": false,
+    "name": "Beaufort",
+    "tabTitle": "Your Tab Title",
+    "fontFace": "Cascadia Mono",
+    "padding": "16, 16, 16, 16",
+    "colorScheme": "Beaufort",
+    "useAcrylic": true,
+    "acrylicOpacity": 0.7
```

Comment:
missing a comma

Reply:
Whoops, that's embarrassing. I chopped the JSON about a little after moving it into the MarkDown file.